scripts/cube_fx.lua

```diff
@@ -310,10 +368,20 @@ local function track_victory_statistics(size, results)
     end
   end
   victory_statistics.cube_last_position = new
+
+  -- Track cube "utilization"
+  local machine_conditional = cube_working_machine_types[entity.type]
+  if machine_conditional and machine_conditional(entity, results[1].item) then
+    -- The cube is in some entity that's using the cube "utilization"
+    victory_statistics.utilization.working = victory_statistics.utilization.working + update_tick_modulus
```

Comment:
You could just add `1` instead of `update_tick_modulus` and get the same result, then we don't need that variable.

Reply:
Yeah true, realized that a few hours after making the PR. Changed to `1`.